src/styles/fixed-card-layout.scss

```diff
@@ -13,46 +13,52 @@ $block: #{$fd-namespace}-fixed-card-layout;
   &__card-layout {
     @include fd-reset();
     @include fd-flex-horizontal-center();
-  }
-
-  &__card-layout-column {
-    @include fd-reset();
-
-    flex: 1;
-    margin-right: $fd-card-margin;
-
-    &:last-child {
-      margin-right: 0;
-
-      @include fd-rtl() {
-        margin-left: 0;
+    @include fd-set-margin-left(-$fd-card-margin);
+
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    justify-content: flex-start;
```

Comment:
On line 15 the `fd-flex-horizontal-center` mixin is imported. This mixin brings:
```
@mixin fd-flex-horizontal-center {
  display: flex;
  justify-content: center;
  @content;
}
```
so you don't need line 18 and you need to put the additional flex related code in the mixin body:
```
@include fd-flex-horizontal-center() {
    flex-wrap: wrap;
    flex-direction: column;
    justify-content: flex-start;
}
```